### DISCUSSION TITLE
Domain issues

### DISCUSSION THREAD

**Discussion by anto2887** [learndevopsonline/learndevopsonline#1559] [Q&A] [answered]
So I got my domain on Sunday and changed the Name servers in the settings to match the ones on my Route53 hosted zone:
![image](https://github.com/learndevopsonline/learndevopsonline/assets/110925506/653f9ae9-44f1-41c5-b2ae-7e7ee6f386e9)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/110925506/467b04d4-dea2-45d0-a7b6-b731c78c1b25)

However when I run a DNS check it seems like the domain is not online:
![image](https://github.com/learndevopsonline/learndevopsonline/assets/110925506/71bef88e-dfcb-40d2-91f4-96def1511130)

I can't seem to figure out what the issue is, when I run the lab to access my front end it takes to long to load and then it fails.

**Comment by sreedharm07** [comment]
In drop down select ns and see your domain was active

**Reply by anto2887** [reply]
it's active:
![image](https://github.com/learndevopsonline/learndevopsonline/assets/110925506/79eaf130-717f-470b-937f-06a956341dd9)

However I am still having issues accessing my frontend with the record created using the private IPv4 address.

**Reply by prabhavith** [reply]
Give public ip of frontend in ur browser

**Reply by sreedharm07** [reply]
@anto2887 which  record you created i mean A or CName like that ?? 

You can also check your frontend dns in dns checker if it is active there is no problem with your dns record.

**Reply by anto2887** [reply]
@sreedharm07 I created an A record like Raghu created in class and I used the private IPv4 of the servers for it.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/110925506/41e98ea8-0d96-4900-a5a1-821a0c98d909)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/110925506/42aa829b-c2d1-4225-8ab2-bb7e44b2abc6)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/110925506/4dd86aa9-33f7-4b85-a273-cee30f2b308c)

I tried my frontend in dns checker and it's online:
![image](https://github.com/learndevopsonline/learndevopsonline/assets/110925506/d2bdb85a-04f1-47d9-9c9e-a7ee6b78be7b)

**Reply by sreedharm07** [reply]
Can you please send your error screen shot also so that your error can understand easily

**Reply by anto2887** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/110925506/0caf92b3-c3b7-439f-9a17-b1badb663039)
 this is on Firefox

**Reply by sreedharm07** [reply, marked as answer]
Hi @anto2887  
if you want that way give public ip in your dns record,  then you will get UI with your frontend dns name

**Reply by anto2887** [reply]
Oh. I tried that before and it worked. i just thought I was wrong since we used private ips in class. Thanks.

**Comment by anto2887** [comment]
> Give public ip of frontend in ur browser

That works but I want the dns record route to work
